easybuild/easyblocks/c/cuda.py

```diff
@@ -114,3 +148,12 @@ def make_module_req_guess(self):
         })
 
         return guesses
+    
+    def make_module_extra(self):
+        """Specify CUDA_HOME """
+
+        txt = super(EB_CUDA, self).make_module_extra()
+
+        txt += self.module_generator.set_environment('CUDA_HOME', self.installdir)
```

Comment:
@damianam: the guesses are checked, in the sense that `eb` will check whether the directory actually exists before including the corresponding statement in the module file.

Additionally, via guesses we'll use `prepend-path` rather than `setenv`.

For `$CUDA_HOME` in particular, I'm not sure if `prepend-path` makes sense at all. Probably not...